### ISSUE TITLE
Sourcemaps on Every Error?

### ISSUE THREAD

**Issue by trisys3** [Kuniwak/gulp-csslint-sourcemap-reporter#1]
Why is this plugin creating a sourcemap on every error? It would be much more efficient to create the sourcemap just once, then use the same sourcemap on every error, since we are running this plugin on a single file each time. I clocked this as taking 4 times as long to run than compiling with sourcemaps (with LESS) then running CSSLint.

**Comment by Kuniwak** [comment]
> Why is this plugin creating a sourcemap on every error?

No, the sourcemap are created once on Less plugin or Sass plugin.

But this plugin create a sourcemap consumer on every error. It is not efficient.
I'm gonna fix it, but I think this fix is only lightly affected to performance.

So, I don't know the cause.
I'm gonna measure on twitter bootstrap's less files.
